helmfile/values/prometheus-blackbox-exporter-wc.yaml.gotmpl

```diff
@@ -4,3 +4,42 @@ config:
   modules:
     tcp_connect:
       prober: tcp
+
+serviceMonitor:
+  enabled: true
+  targets:
+    {{- if .Values.prometheusBlackboxExporter.targets.prometheus }}
+    - name: prometheus
+      url: kube-prometheus-stack-prometheus.monitoring.svc.cluster.local:9090
+      interval: 60s
+      scrapeTimeout: 30s
+      module: http_2xx
+    {{- end }}
+    {{- if .Values.prometheusBlackboxExporter.targets.rook }}
+    - name: rook
+      url: rook-ceph-mgr.rook-ceph.svc.cluster.local:9283
+      interval: 60s
+      scrapeTimeout: 30s
+      module: http_2xx
+    {{- end }}
+    {{- if .Values.prometheusBlackboxExporter.targets.nginx }}
+    - name: nginx
+      url: ingress-nginx-controller-admission.ingress-nginx.svc.cluster.local:443
```

Comment:
Shouldn't you be able to use the `http_2xx` for this one? Same for gatekeeper

Reply:
Nope, you can test it by running: 
compliantkubernetes-apps/bin/ck8s ops kubectl wc port-forward svc/prometheus-blackbox-exporter -n monitoring 9115
http://localhost:9115/probe?target=ingress-nginx-controller-admission.ingress-nginx.svc.cluster.local:443&module=tcp_connect&debug=true

Some of the services require "tcp_connect" it seems like.